tools/RadioGUI/gui.py

```diff
@@ -121,6 +121,12 @@ def process(line):
                 console.insert(END, msg + "\n")
                 logging.info("Decoded Telem: " + msg)
 
+                print("---pyrofets---")
```

Comment:
why this in addition to doing the parser? The goal is that the telem.py has all of the encoding in one place. If the decodes properly into JSON and you just wanted a more visible print, you should print the output from the decoder. Also a better way to display the salient data would be useful, but I never got that far.